eng/Version.Details.xml

```diff
@@ -9,408 +9,408 @@
 -->
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="Microsoft.AspNetCore.Blazor.Mono" Version="5.0.0-alpha1.19516.2">
+    <Dependency Name="Microsoft.AspNetCore.Blazor.Mono" Version="5.0.0-alpha1.19528.2">
```

Comment:
Manually downgraded this, will update it as part of https://github.com/aspnet/AspNetCore/pull/16580